config/enabled.yml

```diff
@@ -265,6 +265,12 @@ Style/FileName:
   StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#snake-case-files'
   Enabled: true
 
+Style/FrozenStringLiteralComment:
+  Description: >-
+                 Add the frozen_string_literal comment to the top of files
+                 to help transition from Ruby 2.3.0 to Ruby 3.0.
+  Enabled: false
```

Comment:
Whoops, it should be in `config/enabled.yml` and set to `true` as per an earlier discussion. I will correct this and submit a PR.